### PR TITLE
[Docs: UI] fixing of the footer social bar

### PR DESCRIPTION
**Pull request by MoeRayo**
**Description**
Fixing of the footer social bar

This PR fixes #
This PR fixes issue #1454 raised by @chandrashritii 
**Notes for Reviewers**
This is the new change effected and it can be seen by scrolling to the footer of the docs and hovering on the icons.
Screenshot_2020-08-27 Meshery Documentation

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by chandrashritii — APPROVED** [review]

**Comment by chandrashritii** [comment]
LGTM. This PR can be merged @leecalcote 

@MoeRayo Good work!

File: docs/_includes/footer.html [hunk 1]
```diff
@@ -56,23 +56,23 @@
...
         <svg xmlns="http://www.w3.org/2000/svg" viewBox="-200 -60 820 740" width="50px" height="50px">
           <path fill="currentColor" stroke="currentColor" stroke-width="3" stroke-miterlimit="10" d="M336 292v24c0 6.6-5.4 12-12 12h-76v76c0
-		  6.6-5.4 12-12 12h-24c-6.6 0-12-5.4-12-12v-76h-76c-6.6 0-12-5.4-12-12v-24c0-6.6 5.4-12 12-12h76v-76c0-6.6 5.4-12 12-12h24c6.6 0 12 
+		  6.6-5.4 12-12 12h-24c-6.6 0-12-5.4-12-12v-76h-76c-6.6 0-12-5.4-12-12v-24c0-6.6 5.4-12 12-12h76v-76c0-6.6 5.4-12 12-12h24c6.6 0 12
 		  5.4 12 12v76h76c6.6 0 12 5.4 12 12zm112-180v352c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V112c0-26.5 21.5-48 48-48h48V12c0-6.6
 		  5.4-12 12-12h40c6.6 0 12 5.4 12 12v52h128V12c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v52h48c26.5 0 48 21.5 48 48zm-48 346V160H48v298c0
 		  3.3 2.7 6 6 6h340c3.3 0 6-2.7 6-6z"/>
         </svg>
 
-        <span class="tooltiptext">Subscribe to the community calender</span></a>
```
Comment (leecalcote) [inline docs/_includes/footer.html:73]: Thanks for this PR, @MoeRayo. 

If we are creating a new CSS class, are we removing the old class?

**Comment by leecalcote** [comment]
I realize this issue was already present, but while we're cleaning up the footer, @MoeRayo will you change the fill color of the tooltip to be one of the these two colors, so that legibility is improved? - #477E96 or #647881

These colors come from https://layer5.io/brand.

Screen Shot 2020-08-27 at 9 16 13 AM

File: docs/_includes/footer.html (hunk 1 above)
Comment (MoeRayo) [reply docs/_includes/footer.html:73]: The other icons had the class of `footer-icons-text` except that icon and I figured that must be where the bug was coming from, so I changed the class to match the other icons present.

Comment (leecalcote) [reply docs/_includes/footer.html:73]: Oh, I see. Sounds real good.

**Comment by MoeRayo** [comment]
@leecalcote I just made the change you requested and I used #477E96 instead

**Comment by leecalcote** [comment]
Screen Shot 2020-08-27 at 10 55 58 AM

Nice work, @MoeRayo 👍

**Pull request merged**